### ISSUE TITLE
Purifycss choking on inline assets

### ISSUE THREAD

**Issue by MilosRasic** [webpackmonitor/webpackmonitor#50]
We have assets urls in CSS converted into data url by url-loader and svg-url-loader. Purifycss is choking on these data urls with message:

Unhandled rejection Error: undefined:2933:467: missing '}'

This seems to be the same thing as https://github.com/purifycss/purifycss/issues/144

Does webpack monitor even need to use purifycss? If the user is not using purifycss in their own setup, why use it when generating stats?

**Comment by roachjc** [comment]
Fair point ... I should set CSS checking as optional, as that and the minify checker are a pretty trivial. Was more a proof of concept thing for more useful evaluation of assets. Will put on the list.

**Comment by MilosRasic** [comment]
Actually, I figured out it's only checking the purified size for the recommendations page. My workaround was to wrap the purifycss call in a try...catch and display a message that purifycss couldn't parse if it catches. I'll make a PR later today or tomorrow so you can choose between that or another approach.

**Comment by roachjc** [comment]
That would be awesome. Even if we make the recommendations optional, better error handling is still required so what you suggest would be much appreciated. Thank you!
